src/Math-Complex-Number/PMComplexNumber.class.st

```diff
@@ -80,7 +80,7 @@ Class {
 		'real',
 		'imaginary'
 	],
-	#category : #'Math-Complex'
+	#category : #'Math-Complex-Number-Math-Complex-Number'
```

Comment:
Math-Complex-Number-Math-Complex-Number ??

Reply:
Oops. I'll sort that out for us. Great spot